<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>982</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addCard">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加ic卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readCard">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>读取ic卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearCard">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空ic卡</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textLog">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>256</width>
     <height>831</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getHotelInfo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>获取hotelInfo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startSocket">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>760</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>177</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>楼层号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="floor">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="connect">
      <property name="text">
       <string>连接发卡器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="onconnect">
      <property name="text">
       <string>断开连接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>177</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>楼栋号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="building">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>189</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>有效时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="time">
      <property name="text">
       <string>1684344289</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>370</y>
     <width>183</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>设备MAC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="doorMac">
      <property name="text">
       <string>D059F21EE252</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>appid</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="appid">
      <property name="text">
       <string>f37d200d9d5b4250924ce43280ed806e</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>secrets</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="secrets">
      <property name="text">
       <string>4b88327993e0695ea16d54be377a639b</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QPushButton" name="writeHotelCard">
      <property name="text">
       <string>写酒店专用卡</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="emptyCard">
      <property name="text">
       <string>恢复空白卡</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>640</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>socket端口号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="socketPort">
      <property name="text">
       <string>8989</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>发卡器串口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cardPort">
      <property name="text">
       <string>COM3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
